packages/interface/src/components/navigable-region/README.md

```diff
@@ -29,10 +27,10 @@ A meaningful name for the ARIA landmark region.
 -   Type: `String`
 -   Required: Yes
 
-### motionProps
+### as
 
-Properties of `motionProps` object will be used by the `motion` component to set a CSS animation on the wrapper div.
+The component used as the root of the region. Defaults to the `div` element.
```

Comment:
Not an expert, just a couple questions:
- should we document it's a Component or HTML element?
- Maybe `Defaults to the`  should be changed to `Defaults to a`